tgui/packages/tgui/interfaces/Oven.tsx

```diff
@@ -0,0 +1,218 @@
+/**
+ * @file
+ * @copyright 2025
+ * @author FlameArrow57 (https://github.com/FlameArrow57)
+ * @license ISC
+ */
+
+import {
+  Button,
+  Divider,
+  Icon,
+  LabeledList,
+  Modal,
+  Section,
+  Stack,
+} from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+interface PacketInfo {
+  time: number;
+  heat: string;
+  cooking: boolean;
+  content_icons: Array<Object>;
+  content_names: Array<string>;
+  recipe_icons: Array<Object>;
+  recipe_names: Array<string>;
+  output_icon: Object;
+  output_name: string;
+  cook_time: string;
+}
+
+export const Oven = () => {
+  const { act, data } = useBackend<PacketInfo>();
+  const {
+    time,
+    heat,
+    cooking,
+    content_icons,
+    content_names,
+    recipe_icons,
+    recipe_names,
+    output_icon,
+    output_name,
+    cook_time,
+  } = data;
+
+  return (
+    <Window title="Cookomatic Multi-Oven" width={420} height={555}>
+      <Window.Content>
+        <Section
+          title="Settings"
+          buttons={
+            <Button
+              onClick={() => act('open_recipe_book')}
+              tooltip={'Warning: Slow'}
+            >
+              Open Recipe Book
+            </Button>
+          }
+        >
+          <Stack fontSize={1.5} justify="center">
+            <Stack.Item vertical>
+              <Stack.Item>
+                <OvenDialbuttons min={1} max={5} time={time} />
+              </Stack.Item>
+              <Stack.Item>
+                <OvenDialbuttons min={6} max={10} time={time} />
+              </Stack.Item>
+            </Stack.Item>
+            <Stack.Item vertical>
+              <Stack.Item>
+                <Button
+                  color={heat === 'High' ? 'red' : 'grey'}
+                  onClick={() => act('set_heat', { heat: 'High' })}
+                  minWidth="75px"
+                  textAlign="center"
+                >
+                  High
+                </Button>
+              </Stack.Item>
+              <Stack.Item mt={0.5}>
+                <Button
+                  color={heat === 'Low' ? 'blue' : 'grey'}
+                  onClick={() => act('set_heat', { heat: 'Low' })}
+                  minWidth="75px"
+                  textAlign="center"
+                >
+                  Low
+                </Button>
+              </Stack.Item>
+            </Stack.Item>
+          </Stack>
+          <Stack justify="center">
+            <Stack.Item>
+              <Button selected onClick={() => act('start')} fontSize={2} mt={0.5}>
```

Comment:
This button shouldn't be selected as that doesn't mean anything in this context.